OtripleS.Web.Api.Tests.Unit/Services/Foundations/Users/UserServiceTests.Validations.RetrieveById.cs

```diff
@@ -18,8 +18,8 @@ public partial class UserServiceTests
         public async Task ShouldThrowValidatonExceptionOnRetrieveWhenUserIdIsInvalidAndLogItAsync()
         {
             // given
-            Guid randomUserId = default;
-            Guid inputUserId = randomUserId;
+            Guid invalidUserId = Guid.Empty;
+            Guid inputUserId = invalidUserId;
```

Comment:
Remove the input variable

Reply:
@ShriHumrudha When I remove the Guid inputUserId = invalidUserId; It fails to build. Am I removing the whole variable or should I change it to Guid someUserId = invalidUserId;?

Reply:
Use invalidUserId instead of inputUserId then removing inputUserId will not have any failures.